src/data.js

```diff
@@ -94,6 +94,7 @@ export const getAllData = async (list, all = false) => {
     data.states = statesData;
   }
 
+<<<<<<< HEAD
```

Comment:
Not sure what happened here but if you could fix the merge conflict text that would be great

Reply:
Oh sorry. @mahowa, I will fix it